strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/login/LoginController.java

```diff
@@ -35,7 +36,7 @@
 import static org.carlspring.strongbox.users.security.JwtAuthenticationClaimsProvider.JwtAuthentication;
 
 /**
- * Works in conjunction with {@link org.carlspring.strongbox.security.authentication.suppliers.JsonFormLoginSupplier}
+ * Works in conjunction with {@link JsonFormLoginStrategy}
```

Comment:
Could you please use the full class name here?

Reply:
Done